build.gradle

```diff
@@ -14,48 +14,59 @@ plugins {
     id 'idea'
 }
 
+// Define tool and dependency versions
```

Comment:
Any reason this is here and the `dependencies.gradle` is removed?

Reply:
I found it a bit confusing at first to have the dependency block in `build.gradle` and the actual definition of the dependency in the `dependencies.gradle`. My priority was here to have all the version numbers all in a common place for easier updating.
As long as there is only one dependency, I think it's not really needed to spread the build script over multiple files.
In the end it boils down to personal preference. If you'd like, we could keep the `dependencies.gradle`, but then maybe also move the `dependencies {}` block over there and as much of the version numbers as easily possible.